src/components/Semester.vue

```diff
@@ -99,20 +100,9 @@
             />
           </div>
         </div>
-        <div
-          class="semester-courseWrapper semester-addWrapper"
-          :class="{ 'semester-addWrapper--compact': compact }"
-          @click="openCourseModal"
-          data-intro-group="pageTour"
-          data-step="3"
-          data-intro='<b>Add your course in this semseter!</b><br>
-            <div class = "introjs-bodytext">To start planning your college career, you should try adding a course in your current semester.</div>'
-          data-disable-interaction="1"
-        >
-          <span class="semester-buttonText" :class="{ 'semester-buttonText--compact': compact }">{{
-            buttonString
-          }}</span>
-        </div>
+        <addcoursebutton :compact="compact" @click="openCourseModal">
+          {{ buttonString }}
```

Comment:
nice factoring out the AddCourseButton! Given the name I feel like you shouldn't have to input the `buttonString` "+ Course" and it should just be hardcoded into the `AddCourseButton.vue` file?

Reply:
That's fair cause I don't think it will be anything else